packages/local_auth/local_auth_darwin/darwin/Classes/FLALocalAuthPlugin.m

```diff
@@ -153,6 +153,26 @@ - (void)showAlertWithMessage:(NSString *)message
           dismissButtonTitle:(NSString *)dismissButtonTitle
      openSettingsButtonTitle:(NSString *)openSettingsButtonTitle
                   completion:(FLADAuthCompletion)completion {
+#if TARGET_OS_OSX
+  NSAlert *alert = [[NSAlert alloc] init];
+  [alert setMessageText:message];
+  [alert addButtonWithTitle:dismissButtonTitle];
+  if (openSettingsButtonTitle != nil) {
+    [alert addButtonWithTitle:openSettingsButtonTitle];
+  }
+  [alert beginSheetModalForWindow:NSApp.keyWindow
```

Comment:
See https://github.com/flutter/packages/pull/5766/files#r1486389711 for discussion of the window that should be used; there is no guarantee that the Flutter content is in the key window.

Reply:
I am not seeing how to get the current window with the registrar. Can you link me to some documentation please?

Reply:
Nvm I found it

Reply:
NSWindow* keyWindow = self.registrar.view.window;